### PR TITLE
Hypno dev

### PR DESCRIPTION
**Pull request by Hypno-Milk**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/Hypno-Milk/keymap.c [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+      #endif
+    } else {
+      #ifdef MOUSEKEY_ENABLE
+        tap_code(KC_MS_WH_UP);
+      #else
+        tap_code(KC_PGUP);
+      #endif
+    }
+  }
+    return true;
```
Comment (drashna) [inline keyboards/planck/keymaps/Hypno-Milk/keymap.c:309]:
```suggestion
    return false;
```

**Review by drashna — CHANGES_REQUESTED** [review]
Please rename the folders to be lower case, and remove the "qmk_firmware" submodule

**Comment by Hypno-Milk** [comment]
Apologies Drashna, 

I wasn’t aware I had tried submitting changes to the main repo - I’m still very new to git and how it works.

I’ll figure out how to cancel this request once home, until I have my actual key map properly complete and ready to upload.

And I’d like to say a quick thank you, I recognise your name from several guides and comments that I’ve used while learning about qmk in general. Thank you for your time, effort and shared knowledge!

Cheers,

**Comment by drashna** [comment]
no worries!

I've closed the pull request for you, so no worries.

**Pull request closed without merging**